AttributeError: 'Version' object has no attribute 'release'

Quick headsup as I am fairly confident this isnt a problem with my code.  Possibly a problem with my `packaging` version

```py
constr = '~>0.4.0'

    @staticmethod
    def _parse_npm(constr: str) -> Set[Specifier]:
        version = parse(constr.lstrip('~^=>').replace('.*', '.0'))
        if isinstance(version, LegacyVersion):
            raise InvalidSpecifier(constr)
>       parts = version.release + (0, 0)
E       AttributeError: 'Version' object has no attribute 'release'
```
```
> python3 -c 'import packaging; print(packaging.__version__)'
19.0
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,4 +24,4 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.5"
-packaging = "*"
+packaging = ">=17.1"

